lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -14,6 +14,15 @@ def stub_publishing_api_put_content(content_id, body)
         stub_publishing_api_put(content_id, body, '/content')
       end
 
+      def stub_publishing_api_put_content_and_return_response(content_id, response)
```

Comment:
What is the reason that you're choosing to add specific methods for a stub that returns something?

For example, this method could be created by adding a third argument to `#stub_publishing_api_put_content`.